src/content/drafts/eslint-community-org.md

```diff
@@ -0,0 +1,131 @@
+  We took this one over from the
+  [@nodesecurity](https://github.com/nodesecurity) team, but
+  [@nzakas](https://github.com/nzakas) is still maintaining it
+* [`eslint-utils`](https://github.com/eslint-community/eslint-utils) &
+  [`regexpp`](https://github.com/eslint-community/regexpp)  
+  We created a fork from [@mysticatea](https://github.com/mysticatea)'s repos &
+  published them under the
+  [`@eslint-community` npm Registry organization](https://npmjs.com/org/eslint-community).
+
+We also intend to fork, bring up to date, and publish the following packages
+originally authored by [@mysticatea](https://github.com/mysticatea) as well:
+
+* [`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es)  
+  We'll work on getting all
+  [`eslint-plugin-es-x`](https://github.com/eslint-community/eslint-plugin-es-x)
+  features merged as well
+* [`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments)
+* [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node)  
+  We'll work on getting all
+  [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
+  features merged as well
+
+Of course, we don't want to stop there.
+
+If you think your project would be a good fit for the new community org, *or*
+you're depending on a project that's currently unmaintained & want to have the
+community to take it over, you can contact the Community Core team on
+[the official ESLint Discord](https://eslint.org/chat).
+
+Keep in mind that we'll only be able to take over projects that are adhering to
+the following acceptance criteria:
+
+* Is it a package that is ESLint-related?  
+  Mostly this will be ESLint plugins, but (unmaintained) dependencies of such
+  packages, closely related packages or packages split from the main ESLint repo
+  (like
+  [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  or
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table))
+  or used by the main repo (like
+  [`eslint-utils`](https://github.com/mysticatea/eslint-utils) &
+  [`regexpp`](https://github.com/mysticatea/regexpp)) could go in the
+  `@eslint-community` GitHub organization as well.
+
+  Because plugins can expose their own (recommended) configs, we already accept
+  config packages in some way.  
+  However, the community core team won't be eager to accept shared ESLint config
+  packages, as they can be very opinionated.
+
+  It also doesn't matter if the package is purely TypeScript-related or not, as
+  we don't want to fragment the ESLint community more without any real benefit.
+
+* Is it widely depended upon throughout the ESLint community?  
+  We don't have a real number in mind here, but the packages we currently took
+  over (or that we're planning to take over) almost all have at least 3M
+  downloads/week.
+
+* Is it currently unmaintained?  
```

Comment:
I took the wording we used in the RFC here

Reply:
The RFC was designed to get a starting point for the project, but we definitely discussed that this should be an org for more than just orphaned projects. There was a lot of discussion about who would be handling what if maintainers were still in the picture. If that's not what you're intending now, then we need to have a larger discussion because it seems that our two ideas for this org have diverged significantly and we need to fix that.

Reply:
I don't mind having non-orphaned projects into the org, but I think I missed the discussions you are talking about 🤔
Hence why I only included the text about orphaned projects.

Feel free to fill me in on the outcome of them or point me to them if they're available publicly

Reply:
I mentioned this from the start:
https://github.com/eslint/eslint/discussions/15929#discussioncomment-2850464

I think we both probably weren't terribly clear when communicating this, as it seems like there are several places where you mentioned "unmaintained" whereas I was mentioning "in case they become unmaintained." And it seems like we never fully clarified our positions there.

There was also discussion on the RFC about inviting maintainers to continue working on the project as part of the org:

https://github.com/eslint/rfcs/pull/91#issuecomment-1145333115 - here, I was assuming that maintainers would still be maintaining their packages.
https://github.com/eslint/rfcs/pull/91#discussion_r899571650 - it seems like you were assuming the same here.

The RFC itself mentions, in several places, that not being maintained isn't a hard-and-fast rule:

https://github.com/eslint/rfcs/pull/91/files#diff-fe2bbc1acd365b32adaeb59fea2c39b87fb9974c30902c0342ecddfa13355138R110
https://github.com/eslint/rfcs/pull/91/files#diff-fe2bbc1acd365b32adaeb59fea2c39b87fb9974c30902c0342ecddfa13355138R157

So, I think we were probably just talking past each other a bit throughout the whole process. It happens, not a big deal.

What I would like to say going forward is that projects that meet the criteria besides no updates will be accepted. As we've learned through mysticatea's plugins, trying to regain control of plugins after they have fallen out of maintenance is a lot of work. It would be a lot easier if those plugins were already in eslint-community, being maintained by their original maintainers, and then we could step in if something happened to them. That's exactly why I transferred eslint-plugin-security and why I would like other plugin authors to do the same.

How does that sound?